core/genesis.go

```diff
@@ -616,6 +616,18 @@ func DefaultOdysseyGenesisBlock() *Genesis {
 	}
 }
 
+// DefaultStoryGenesisBlock returns the story mainnet genesis block.
+func DefaultStoryGenesisBlock() *Genesis {
+	return &Genesis{
+		Config:     params.StoryChainConfig,
+		Difficulty: big.NewInt(0x20000),
+		GasLimit:   0x7A1200,
+		Nonce:      0x42,
+		Timestamp:  0,
+		Alloc:      decodePrealloc(odysseyAllocData), // TODO: change to Story mainnet alloc data
```

Comment:
Can we copy odysseyAllocData to storyAllocData

Reply:
Done in [15ff3fe](https://github.com/piplabs/story-geth/pull/77/commits/15ff3fef8666f1715bab9d27d81f69eb736c2488).